Xamarin.Forms.Platform.Android/Renderers/ButtonDrawable.cs

```diff
@@ -36,6 +36,9 @@ float PaddingTop
 			set { _paddingTop = value; }
 		}
 
+		double BorderWidth => Math.Max(Button.IsSet(Button.BorderWidthProperty) ? Button.BorderWidth : .25, 0);
+		Color BorderColor => Button.IsSet(Button.BorderColorProperty) && Button.BorderColor != Color.Default ? Button.BorderColor : Color.FromHex("#c5c5c5");
```

Comment:
Same here - where do these colors come from? Is there a guide or source code we can point to?